src/open_samus_returns_rando/files/templates/randomizerpowerup.lua

```diff
@@ -96,6 +97,34 @@ function RandomizerPowerup.MarkLocationCollected(actorOrName)
     Blackboard.SetProp(playerSection, propName, "b", true)
 end
 
+local AeionAbilityClouds = {
```

Comment:
`local` on file level doesn't make much sense, does it? I would just move it into the function because there is no reason to make it global.